src/main/java/com/thealgorithms/others/SpiralMatrixII.java

```diff
@@ -0,0 +1,49 @@
+package com.thealgorithms.others;
+
+import java.util.Arrays;
+
+public class SpiralMatrixII {
+    public int[][] generateMatrix(int n) {
```

Comment:
```suggestion
    static public int[][] generateMatrix(int n) {
```
I would l suggest to rename the variable `n` to something like `size`.

What happens if `n` is negative?

Reply:
okay i will look after it